mailbox/store/src/test/java/org/apache/james/mailbox/store/probe/SieveProbeTest.java

```diff
@@ -0,0 +1,23 @@
+package org.apache.james.mailbox.store.probe;
+
+import org.junit.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class SieveProbeTest {
+
+    @Test
+    public void testSanitizeFileNameWithValidInputForSieveProbe() {
+        String validScriptFileNameSiv = "myscript.siv";
+        String validScriptFileNameSieve = "myscript.sieve";
+        assertThat(SieveProbe.sanitizeScriptName(validScriptFileNameSiv)).isEqualTo(validScriptFileNameSiv);
+        assertThat(SieveProbe.sanitizeScriptName(validScriptFileNameSieve)).isEqualTo(validScriptFileNameSieve);
```

Comment:
You should not test two things in a single test

Reply:
I will separate it in two test cases.

Reply:
Done.